src/Sarif.Viewer.VisualStudio/Converters/ImportanceToBackgroundConverter.cs

```diff
@@ -0,0 +1,27 @@
+﻿using Microsoft.CodeAnalysis.Sarif;
```

Comment:
Can you add the copyright/license comment to this file. Just copy it from one of the other files. 
Also, do the same for ImportanceToForegroundConverter.cs, if required.